vitalam-node-api/src/env.js

```diff
@@ -0,0 +1,19 @@
+const envalid = require('envalid')
+const dotenv = require('dotenv')
+
+if (process.env.NODE_ENV === 'test') {
+  dotenv.config({ path: 'test/test.env' })
+} else {
+  dotenv.config({ path: '.env' })
+}
+
+const vars = envalid.cleanEnv(process.env, {
+  LEGACY_METADATA_KEY: envalid.str({ default: '' }),
```

Comment:
I think you can remove this env (and in README)? It's not used in the types definition anymore and will soon be completely gone from API with my current ticket to remove legacy code